### ISSUE TITLE
✅ 🐞 Function print_emoji_styles is deprecated since version 6.4.0! Use wp_enqueue_emoji_styles instead.

### ISSUE THREAD

**Issue by jamieburchell** [elementor/elementor#27502]
### Prerequisites

- [X] I have searched for similar issues in open and closed tickets and cannot find a duplicate.
- [X] I have troubleshooted my issue, and it still exists against the latest stable version of Elementor.

### Description

As per https://github.com/elementor/elementor/issues/27307.

> Elementor does not use this function. This is not coming from Elementor

While Elementor does not use this function, it appears to be interfering with its removal from the queue or is not removing it where it should to avoid the deprecation message. Two points of reference are [wp-includes/formatting.php:L5872](https://github.com/WordPress/WordPress/blob/5359baba0a62a4fd61bcab52f519e6c6e852e2f6/wp-includes/formatting.php#L5871) and an example in the block editor [where the function is removed](https://github.com/WordPress/WordPress/blob/5359baba0a62a4fd61bcab52f519e6c6e852e2f6/wp-includes/block-editor.php#L363) to avoid emitting the deprecation message.

> Please, troubleshoot your issue before submitting it here,

This is consistently reproducible with *only* Elementor active in WordPress 6.5.3 (not even Pro is required to repro) and I *only* encounter it when using a function of Elementor.

### Steps to reproduce

1. PHP 8.2 / WordPress 6.5.3
2. Only Elementor plugin active
3. Click "Theme Builder"
4. Look at debug log for deprecation

### Expected behavior

WordPress can dequeue the affected function as normal

### Isolating the problem

- [X] This bug happens when only the Elementor (and Elementor Pro) plugins are active.
- [X] This bug happens with the Hello Elementor theme active.
- [X] I can reproduce this bug consistently by following the steps I described above.

### Elementor System Info

<details>
<summary>Click to reveal</summary>

````txt

== Server Environment ==
	Operating System: Linux
	Software: Apache/2.4.52 (Ubuntu)
	MySQL version: Ubuntu 22.04 v10.6.16
	PHP Version: 8.2.19
	PHP Memory Limit: 256M
	PHP Max Input Vars: 10000
	PHP Max Post Size: 64M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.5.3
	Site URL: https://example.com/wp
	Home URL: https://example.com
	WP Multisite: No
	Max Upload Size: 16 MB
	Memory limit: 40M
	Max Memory limit: 256M
	Permalink Structure: /%category%/%year%/%monthnum%/%day%/%postname%/
	Language: en_GB
	Timezone: Europe/London
	Debug Mode: Active

== Theme ==
	Name: Hello Elementor
	Version: 3.0.1
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en-GB
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/125.0.0.0 Safari/537.36

== Active Plugins ==
	Elementor
		Version: 3.21.8
		Author: Elementor.com

	Elementor Pro
		Version: 3.21.3
		Author: Elementor.com


== Must-Use Plugins ==
	Bedrock Autoloader
		Version: 1.0.3
		Author: Roots

	Disable status tests
		Version: 1.0.0
		Author: Roots

	Register Theme Directory
		Version: 1.0.0
		Author: Roots


== Features ==
	Custom Fonts: 0
	Custom Icons: 0

== Integrations ==
	


== Elementor Experiments ==
	Improved Asset Loading: Active by default
	Improved CSS Loading: Active
	Inline Font Icons: Inactive
	Additional Custom Breakpoints: Inactive
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Active
	Upgrade Swiper Library: Active
	Grid Container: Inactive
	Nested Elements Performance: Inactive by default
	Optimized Control Loading: Inactive
	Default to New Theme Builder: Active
	Hello Theme Header & Footer: Inactive
	Elementor Home Screen: Active by default
	Editor Top Bar: Inactive
	Build with AI: Inactive
	Landing Pages: Inactive
	Nested Elements: Inactive
	Lazy Load Background Images: Inactive
	Display Conditions: Inactive
	Form Submissions: Active
	Menu: Inactive
	Taxonomy Filter: Inactive


== Log ==
	
Log: showing 20 of 402024-04-30 11:45:51 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.21.0',
  'to' => '3.21.1',
)]
2024-05-07 10:26:52 [info] Update DB has been started [array (
  'plugin' => 'Elementor',
  'from' => '3.21.3',
  'to' => '3.21.4',
)]
2024-05-07 10:26:52 [info] elementor::elementor_updater Started 
2024-05-07 10:26:52 [info] Elementor/Upgrades - _on_each_version Start  
2024-05-07 10:26:52 [info] Elementor/Upgrades - _on_each_version Finished 
2024-05-07 10:26:52 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.21.3',
  'to' => '3.21.4',
)]
2024-05-07 10:26:55 [info] Update DB has been started [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.21.1',
  'to' => '3.21.2',
)]
2024-05-07 10:26:55 [info] elementor-pro::elementor_pro_updater Started 
2024-05-07 10:26:55 [info] Elementor Pro/Upgrades - _on_each_version Start  
2024-05-07 10:26:55 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2024-05-07 10:26:55 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.21.1',
  'to' => '3.21.2',
)]
2024-05-13 09:24:28 [info] Update DB has been started [array (
  'plugin' => 'Elementor',
  'from' => '3.21.4',
  'to' => '3.21.5',
)]
2024-05-13 09:24:28 [info] elementor::elementor_updater Started 
2024-05-13 09:24:28 [info] Elementor/Upgrades - _on_each_version Start  
2024-05-13 09:24:28 [info] Elementor/Upgrades - _on_each_version Finished 
2024-05-13 09:24:28 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.21.4',
  'to' => '3.21.5',
)]
2024-05-26 15:33:43 [info] elementor::elementor_updater Started 
2024-05-26 15:33:43 [info] Elementor/Upgrades - _on_each_version Start  
2024-05-26 15:33:43 [info] Elementor/Upgrades - _on_each_version Finished 
2024-05-26 15:33:43 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.21.5',
  'to' => '3.21.8',
)]

PHP: showing 3 of 3PHP: 2024-05-15 07:22:43 [warning X 2][../plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php::150] Undefined array key "condition_type" [array (
  'trace' => '
#0: ../plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(150): Elementor\Core\Logger\Manager -> rest_error_handler()
#1: ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> normalize_template_json_item()
#2: ../plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(120): class type array_map()
#3: ../plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(59): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> normalize_templates_json()
#4: ../plugins/elementor/data/base/endpoint.php(158): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> get_items()
',
)]
PHP: 2024-05-26 15:35:03 [warning X 8][../plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php::150] Undefined array key "condition_type" [array (
  'trace' => '
#0: ../plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(150): Elementor\Core\Logger\Manager -> rest_error_handler()
#1: ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> normalize_template_json_item()
#2: ../plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(120): class type array_map()
#3: ../plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(59): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> normalize_templates_json()
#4: ../plugins/elementor/data/base/endpoint.php(158): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> get_items()
',
)]
PHP: 2024-05-26 15:41:39 [warning X 1][../plugins/elementor-pro/modules/screenshots/module.php::41] Cannot modify header information - headers already sent by (output started at ../plugins/elementor-pro/modules/screenshots/module.php:33) [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]



== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatible

== Elementor Pro - Compatibility Tag ==

````

</details>

### Agreement

- [X] I agree that my issue may be closed without action if it doesn't meet all the requirements.

**Comment by DesignerElise** [comment]
any update?

**Comment by tableconnect** [comment]
same problem: since updating wordpress today. every time we open elementor theme builder, this warning/error message appears for a second before theme builder is loaded. (after that everything seems to operate as expected but still something is not right...)

**Comment by nicholaszein** [comment]
Hey @jamieburchell!

We have great news! 🙌 

### 📢 We're happy to announce that thanks to your feedback this issue was resolved with the release of Elementor Pro v3.26.0! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂

----
PS: If you still experience the same issue after updating to the latest version, and can reproduce it consistently even after troubleshooting it and clearing all levels of cache, please [open a new Bug Report](https://elemn.to/gh-new-bug-report) with detailed information so we can investigate it further.
